### ISSUE TITLE
Improved Mount Management & Configuration

### ISSUE THREAD

**Issue by YPyltiai** [magento/magento-cloud-docker#48]
Internal ID: `MAGECLOUD-3248`

**As** a Magento developer, **I want** local dev solution to provide more granular controls over mounts and file system, **so** I can easily configure mounts without side effects on performance.

**To Do:**
* Split `/var/www/magento/pub` volume to `/var/www/magento/pub/static` and `/var/www/magento/pub/media`
* Add more control over mounts (ie - possibility to modify the list of volumes)
```
      - /var/www/magento/vendor
      - /var/www/magento/generated
      - /var/www/magento/pub
      - /var/www/magento/var
```
* Consider adding possibility to configure mounts through `.magento.app.yaml`

**Comment by shiftedreality** [comment]
Internal ticket: https://magento2.atlassian.net/browse/MAGECLOUD-3248

**Comment by YPyltiai** [comment]
Resolved in #103
